<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6B33D2Q" failures="1" tests="5" name="Personnal Loan" time="0.181" errors="0" timestamp="2023-09-04T12:37:30 MDT">
  <testcase classname="test.day1" name="Demo" time="0.05">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.Demo(day1.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.day2" name="PLoan" time="0.002"/>
  <testcase classname="test.day4" name="LoginAPIHome" time="0.005"/>
  <testcase classname="test.day4" name="MobileLoginHome" time="0.002"/>
  <testcase classname="test.day4" name="WebLoginHome" time="0.002"/>
</testsuite> <!-- Personnal Loan -->
